fdbclient/NativeAPI.actor.h

```diff
@@ -255,6 +255,7 @@ class Transaction : NonCopyable {
 	Future< StorageMetrics > waitStorageMetrics( KeyRange const& keys, StorageMetrics const& min, StorageMetrics const& max, StorageMetrics const& permittedError, int shardLimit );
 	Future< StorageMetrics > getStorageMetrics( KeyRange const& keys, int shardLimit );
 	Future< Standalone<VectorRef<KeyRef>> > splitStorageMetrics( KeyRange const& keys, StorageMetrics const& limit, StorageMetrics const& estimated );
+	Future< Standalone<RangeResultRef> > getStorageMetricsList(KeyRange const& keys, int shardLimit);
```

Comment:
`getStorageMetricsList` and `getStorageMetrics` are doing very different things (`getStorageMetrics` is retrieving metrics from storage servers, whereas `getStorageMetricsList` is retrieving metrics from the data distributor), so I would change the name of `getStorageMetricsList` to something like `getDataDistributionMetricsList`.